help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,169 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Connections destionation connection using the Adobe Experience Platform UI.
+---
+# Merkury Enterprise Connections Destination
+
+>[!NOTE]
+>
+>The destination connector and documentation page are created and maintained by the Merkury team. For any inquiries or update requests, please contact your Merkury account representitive.
+
+## Overview
+
+Use the Merkury Enterprise Connections destination to securely deliver audiences to Merkury. Merkury provides marketers easy matching and delivery of person-based audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and ad-tech connections. Merkury is powered by a comprehensive US adult consumer identity graph of 268+ million people.
+
+![A diagram showing the interconnection between Merkury and Experience Platform, including ingestion and activation](../../assets/catalog/data-partners/merkury-connections/media/image1.png)
+
+Follow the steps in this documentation page to create a Merkury Connections destination connection and activate audiences using the Adobe Experience Platform user interface. 
+
+>[!NOTE]
+>
+>If you are looking to activate audiences to media destinations with you Merkury Connect account, use our Merkury Connections destination instead.
```

Comment:
this note is excellent, but it belongs to the Identity document! one additional comment - should be _your_ Merkury account